lib/rucio/tests/test_did_meta.py

```diff
@@ -0,0 +1,69 @@
+# Copyright European Organization for Nuclear Research (CERN)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Asket Agarwal, <asket.agarwal96@gmail.com>
+
+from nose.tools import assert_equal, assert_in, assert_is_instance
+
+from rucio.client.didclient import DIDClient
+from rucio.common.utils import generate_uuid as uuid
+
+
+class TestDidMetaClient():
+
+    def setup(self):
+        self.did_client = DIDClient()
+        self.tmp_scope = 'mock'
+        self.tmp_name = 'name_%s' % uuid()
+        self.did_client.add_did(scope=self.tmp_scope, name=self.tmp_name, type="DATASET")
+
+    def xtest_add_generic_metadata(self):
```

Comment:
you can enable now all of them by removing the "x" at the beginning. testcases starting with "x" are not executed.

once you start adding functionality you will have to add the appropriate assert_equal etc statements here.